assets/scss/carbon/community-theme.scss

```diff
@@ -91,4 +91,14 @@ $logo-size-small: 8rem;
 $icon-size: 0.75rem;
 
 $carbon--theme: $carbon--theme--custom;
-@include carbon--theme();
\ No newline at end of file
+@include carbon--theme();
+
+//== Checkbox
```

Comment:
I like the idea of "tagging" with documentation when we can foresee value for a future refactor, but since I can't identify a clear idea to handle this situations in our current codebase, I'd suggest to maybe open an issue to discuss this more generally.

Otherwise I could be adding "override" here, another contributor could be adding a "TODO", and so on, which wouldn't have a clear benefit for a future refactor of the theme.